### PR TITLE
revise description of TSO variables

### PR DESCRIPTION
**Pull request by en-jin19**
### What is changed, added or deleted? (Required)
Revise a part of unclear description of TSO variables according to comments on https://github.com/pingcap/docs/pull/6770.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -704,7 +704,7 @@ Query OK, 0 rows affected (0.09 sec)
 
 > **注意：**
 >
-> 如果 PD leader 未因 CPU 限制达到瓶颈而导致 TSO RPC 延迟升高，打开 TSO Follower Proxy 可能会导致 TiDB 的语句执行延迟上升，从而影响集群的 QPS 表现。
+> 如果 PD leader 的 TSO RPC 延迟升高，但其现象并非由 CPU 限制达到瓶颈而导致（可能存在网络等问题），此时，打开 TSO Follower Proxy 可能会导致 TiDB 的语句执行延迟上升，从而影响集群的 QPS 表现。
```
Comment (JmPotato) [inline system-variables.md:707]: “CPU 使用率达到瓶颈”比“CPU 限制达到瓶颈”更确切一点。

Comment (en-jin19) [reply system-variables.md:707]:
```suggestion
> 如果 PD leader 的 TSO RPC 延迟升高，但其现象并非由 CPU 使用率达到瓶颈而导致（可能存在网络等问题），此时，打开 TSO Follower Proxy 可能会导致 TiDB 的语句执行延迟上升，从而影响集群的 QPS 表现。
```

File: system-variables.md [hunk 2]
```diff
@@ -1402,7 +1402,7 @@ set tidb_slow_log_threshold = 200;
 
 > **注意：**
 >
-> 如果 PD leader 未因 CPU 限制达到瓶颈而导致 TSO RPC 延迟升高，调高 `tidb_tso_client_batch_max_wait_time` 可能会导致 TiDB 的语句执行延迟上升，影响集群的 QPS 表现。
+> 如果 PD leader 的 TSO RPC 延迟升高，但其现象并非由 CPU 限制达到瓶颈而导致（可能存在网络等问题），此时，调高 `tidb_tso_client_batch_max_wait_time` 可能会导致 TiDB 的语句执行延迟上升，影响集群的 QPS 表现。
```
Comment (en-jin19) [inline system-variables.md:1405]:
```suggestion
> 如果 PD leader 的 TSO RPC 延迟升高，但其现象并非由 CPU 使用率达到瓶颈而导致（可能存在网络等问题），此时，调高 `tidb_tso_client_batch_max_wait_time` 可能会导致 TiDB 的语句执行延迟上升，影响集群的 QPS 表现。
```

**Review by JmPotato — APPROVED** [review]

**Review by TomShawn — APPROVED** [review]

**Pull request merged**